mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -85,6 +85,7 @@ CLASS aih net/minecraft/entity/LivingEntity
 	METHOD cN getOffHandStack ()Lbaz;
 	METHOD cP getSoundVolume ()F
 	METHOD cQ getSoundPitch ()F
+	METHOD cR cannotMove ()Z
```

Comment:
How about `isMotionless` ?

Reply:
That's different. Not being allowed to move is not the same as not currently moving.

Reply:
Yeah, true. `cannotMove` is kind of a weird name though.

Reply:
isImmovable?

Reply:
Not quite, the method returns true if the entity is dead (or sleeping, for players). It doesn't prevent external motion changes.

Reply:
`isStationary` ?

Reply:
That doesn't work. `isStationary` means it's not moving. But it may be moving and have the field set to true (a sheep eating grass while being pushed by water). The purpose of the field is to prevent the AI (or arrow keys, in the case of a player) from moving the player.